.gitlab-ci.yml

```diff
@@ -498,6 +508,67 @@ trigger-contracts-ci:
     - master
     - schedules
 
+#### stage:                        docker
+docker-build-chaos:                &docker-build-chaos
```

Comment:
what was with the reusing the common docker build job as a template for this job?

Reply:
or if it doesn't fit you, make your own template, DIY

Reply:
Im confused about the 'template', what do you mean by 'reusing the common docker build job as a template'?